packages/common/test/core/schemas/internal/follow/FollowCardUiSchema.test.ts

```diff
@@ -0,0 +1,225 @@
+import { CardEditorOptions } from "../../../../../src/core/schemas/internal/EditorOptions";
+import { buildUiSchema } from "../../../../../src/core/schemas/internal/follow/FollowCardUiSchema";
+import { MOCK_CONTEXT } from "../../../../mocks/mock-auth";
+
+describe("buildUiSchema: follow", () => {
+  it("returns the full follow card uiSchema", async () => {
+    const uiSchema = buildUiSchema("", {} as CardEditorOptions, MOCK_CONTEXT);
+
+    expect(uiSchema).toEqual({
```

Comment:
generally speaking, tests that use `.toEqual(...)` with large objects are difficult to manage over time. If one small thing in the structure changes (e.g. the `hub-input-field-gallery-picker` facets), the test will fail with a complex yet vague error message. To fix this failing test, most other devs will simply replace the large object with whatever object is now being passed in _without actually understanding why the test failed_.  

So while it's very efficient to use tests like this, it's not optimal. Instead of doing a full object comparison, consider checking that `elements` exist for a list of scope values, and maybe that those scopes have the expected type of control?

Reply:
Good point! I will update the tests